src/registrar/models/domain.py

```diff
@@ -1397,6 +1415,21 @@ def dns_needed(self):
         logger.info("Changing to DNS_NEEDED state")
         logger.info("able to transition to DNS_NEEDED state")
 
+    def get_state_help_text(self) -> str:
+        """Returns a str containing additional information about a given state.
+        Returns custom content for when the domain itself is expired."""
+
+        if self.is_expired() and self.state != self.State.UNKNOWN:
```

Comment:
This matches the view logic in home.html for "expired". Technically, all "unknown" domains are expired as they don't have a expiration date. Architecturally this is kind of weird that we're excluding unknown because of a display requirement, so we should talk about this at a later point. I think there is a ticket to address this though